drivers/hmis/app/models/hmis/role.rb

```diff
@@ -20,8 +20,8 @@ class Hmis::Role < ::ApplicationRecord
   end
 
   scope :with_any_permissions, ->(*perms) do
-    scope = self
-    perms.map { |p| scope = scope.or(Hmis::Role.where(p => true)) }
+    scope = none
+    perms.each_with_index.map { |p, i| scope = i.zero? ? where(p => true) : scope.or(Hmis::Role.where(p => true)) }
     scope
   end
```

Comment:
I'd suggest using Arel to make this easier ot read. I think the below code is doing  the same thing:
```
rt = Hmis::Role.arel_table
where_clause = perms.map { |perm| rt[perm.to_sym].eq(true) }.reduce(:or)
where(where_clause)
```

Reply:
Fixed